src/guide/quick-start.md

```diff
@@ -48,7 +48,7 @@ If you are unsure about an option, simply choose `No` by hitting enter for now.
 <span class="line"><span style="color:var(--vt-c-green);">&gt; </span><span style="color:#A6ACCD;">npm run dev</span></span>
 <span class="line"></span></code></pre></div>
 
-You should now have your first Vue project running! Here are some additional tips:
+You should now have your first Vue project running! You should note here that the components in the generated app are written in [Composition API](https://vuejs.org/guide/extras/composition-api-faq.html) style. Here are some additional tips:
```

Comment:
What do you think about this?

```suggestion
You should now have your first Vue project running! Note that the example components in the generated app are written using the [Composition API](/guide/introduction.html#composition-api) and `<script setup>`, rather than the [Options API](/guide/introduction.html#options-api). Here are some additional tips:
```